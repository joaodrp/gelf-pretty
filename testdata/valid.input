{"version":"1.1","host":"example.org","short_message":"A short message","timestamp":1385053862.3072,"level":6}
{"version":"1.1","host":"example.org","short_message":"A short message","full_message":"A long message","timestamp":1555004824,"level":7}
{"version":"1.1","host":"example.org","short_message":"A short message","full_message":"Backtrace here\n\nmore stuff","timestamp":1391759952.705,"level":3}
{"version":"1.1","host":"example.org","short_message":"A short message","full_message":"Backtrace here\n\nmore stuff","timestamp":1554919561.6756,"level":1,"_user_id":9001,"_some_info":"foo","_some_env_var":"bar"}
{"version":"1.1","host":"my-server","short_message":"foo","timestamp":1385053862.3072,"level":5,"_app":"my-app","_logger":"my-module"}
