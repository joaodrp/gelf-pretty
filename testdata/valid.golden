[2013-11-21 17:11:02.307] INFO: example.org: A short message 
[2019-04-11 18:47:04.000] DEBUG: example.org: A short message 
	A long message
[2014-02-07 07:59:12.704] ERROR: example.org: A short message 
	Backtrace here
	
	more stuff
[2019-04-10 19:06:01.675] ALERT: example.org: A short message some_env_var=bar some_info=foo user_id=9001
	Backtrace here
	
	more stuff
[2013-11-21 17:11:02.307] NOTICE: my-app/my-module on my-server: foo 
